src/KafkaNET.Library/Requests/FetchRequest.cs

```diff
@@ -56,7 +56,7 @@ public class FetchRequest : AbstractRequest, IWritable
         public const byte DefaultMinBytesSize = 4;
         public const byte DefaultOffsetInfoSizeSize = 4;
 
-        public const short CurrentVersion = 1;
```

Comment:
Major change: Requests would be sent as version 0 so that responses from Kafka 0.8 - 0.9 are parsable